cpp/ql/src/Security/CWE/CWE-570/IncorrectAllocationErrorHandling.ql

```diff
@@ -153,12 +153,38 @@ predicate exprMayThrow(Expr e) {
   )
 }
 
+class NoThrowType extends Struct {
```

Comment:
Qldoc?

Reply:
Thanks. Fixed in 4107e350cb39dee631ad69ab8c4926c3566d94d7.